packages/audio/lame/patches/lame-3.100-am_path_gtk.patch

```diff
@@ -0,0 +1,233 @@
+diff -Naur lame-3.98.4/configure.in lame-3.98.4.patch/configure.in
```

Comment:
lame is not unpacking with that change, also it compiles without, other distros don't carry that change at all so no idea what it fixes

lame is only required by mpd these days

Reply:
small correction: lame is also needed by audioencoder.lame (so kodi can rip audio CDs to mp3) https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/mediacenter/kodi-binary-addons/audioencoder.lame/package.mk#L13

Reply:
ah, and PKG_REV of audioencoder.lame needs to be bumped so that the updated lame dependency is rolled out

haven't checked the other package bumps in detail yet, but any package that is linked statically into some addon will require PKG_REV bumps in the addon

Reply:
dropping lame, raising as a separate PR
-- https://github.com/LibreELEC/LibreELEC.tv/pull/4692